### PR TITLE
posix-time 0.5.2

### PR DESCRIPTION
**Pull request by mwweissmann**

### CODE REVIEW

File: packages/posix-clock/posix-clock.0.4.0-0/url [hunk 1]
```diff
@@ -0,0 +1,2 @@
+http: "https://github.com/mwweissmann/ocaml-posix-clock/archive/0.4.0.tar.gz"
```
Comment (hannesm) [inline packages/posix-clock/posix-clock.0.4.0-0/url]: shouldn't this be `archive: "https://...."`?

File: packages/posix-mqueue/posix-mqueue.0.8.0-0/url [hunk 2]
```diff
@@ -0,0 +1,3 @@
+http:
+  "https://github.com/mwweissmann/ocaml-posix-mqueue/archive/0.8.0.tar.gz"
```
Comment (hannesm) [inline packages/posix-mqueue/posix-mqueue.0.8.0-0/url]: no need for the newline here (and further down)

File: packages/posix-clock/posix-clock.0.4.0-0/url (hunk 1 above)
Comment (mwweissmann) [inline packages/posix-clock/posix-clock.0.4.0-0/url]: changed to 'archive'

File: packages/posix-mqueue/posix-mqueue.0.8.0-0/url (hunk 2 above)
Comment (mwweissmann) [inline packages/posix-mqueue/posix-mqueue.0.8.0-0/url]: changed to 'archive' and removed newline

**Comment by samoht** [comment]
I've re-launched the failing test.

**Pull request merged**